kratos/includes/amatrix_interface.h

```diff
@@ -418,9 +448,67 @@ template <typename T> AMatrix::TransposeMatrix<T> trans(T& TheMatrix){ return AM
 
 template <typename TExpressionType> using vector_expression = AMatrix::MatrixExpression<TExpressionType,AMatrix::row_major_access>;
 
-using ZeroMatrix = AMatrix::ZeroMatrix<double>;
 
-using ZeroVector = AMatrix::ZeroMatrix<double>;
+template <typename TDataType>
```

Comment:
@pooyan-dadvand  just as a comment, this was added here to be able to explicitly remove a constructor that is there in the unmodified Amatrix